src/pages/_app.tsx

```diff
@@ -74,7 +61,6 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
           chains={[...chains, terra2testnet]}
           assetLists={[...assets, terra2testnetAssets]}
           wallets={wallets}
-          endpointOptions={availableChainsEndpoints}
```

Comment:
if we remove this, the wallet will use keplr rpc for sending transaction not from config